src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -81,37 +86,38 @@
 
 <kd-help-section>
   <md-checkbox ng-model="ctrl.isExternal" class="md-primary"
-             ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
+               ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
     Expose service externally
   </md-checkbox>
 </kd-help-section>
 
+<!-- advanced options -->
 <div ng-show="ctrl.isMoreOptionsEnabled()">
   <kd-help-section>
-    <md-input-container>
+    <md-input-container class="md-block kd-deploy-block">
```

Comment:
Actually if we want to follow the pattern it should be something like `kd-deploy-form-input-block` or `kd-deploy-input-block`. Normally we also add name of the page where css is applied or put it in chrome.css without this name if it's something that can be globally used.